src/ert/scheduler/lsf_driver.py

```diff
@@ -444,6 +449,25 @@ async def _get_exit_code_from_bhist(self, job_id: str) -> int:
 
         return LSF_FAILED_JOB
 
+    async def _dump_bhist_job_summary_to_runpath(self, job_id: str) -> None:
+        runpath = self._jobs[job_id].runpath
+        job_name = self._jobs[job_id].name
+        file_path = runpath / f"{job_name}.LSF-bhist-l"
+        bhist_with_args: list[str] = [
+            str(self._bhist_cmd),
+            "-l",  # long format
+            str(job_id),
+        ]
+        process_success, process_message = await self._execute_with_retry(
+            bhist_with_args,
+            retry_codes=(FLAKY_SSH_RETURNCODE,),
+            retries=3,
+            retry_interval=self._sleep_time_between_cmd_retries,
+        )
+        if not process_success:
```

Comment:
Do we want to raise when this failed? As I understand it this should be only some extra info, right?

Reply:
It should not fail, but I can change it to log the error instead.

Reply:
^the execute_with_retry method already logs errors, so I will just ignore it if it fails.